k8s/base/acct-mgt-configmap.yaml

```diff
@@ -1,6 +0,0 @@
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: openshift-acct-mgt-config
-data:
-  identityProvider: developer
```

Comment:
Granted, I decided to put this just as an environment variable in the deployment yaml as it accomplished the same thing and had one less yaml file to maintain.  Having it is is fine as well.  Is there an advantage to this that I am not seeing?

Reply:
I think using a `configGenerator` and environment-style files (like the `config.env file` make things easier to manage. The design allows us to use a single `envFrom` directive, rather than multiple `valueFrom` directives for individual environment variables.

Reply:
...and this also gets us the automatic restarts when we deploy a modified config (because of how Kustomize modifies configmap (and secret) names with a content-specific hash, triggering a redeploy when they change).

Reply:
should we also put the other non-security related environment variables here as well?